tests/baselines/reference/mappedTypeRelationships.errors.txt

```diff
@@ -250,8 +241,6 @@ tests/cases/conformance/types/mapped/mappedTypeRelationships.ts(168,5): error TS
     function f41<T extends Thing>(x: Readonly<Thing>, y: Readonly<T>) {
         x = y;
         y = x;  // Error
-        ~
-!!! error TS2322: Type 'Readonly<Thing>' is not assignable to type 'Readonly<T>'.

```

Comment:
This error should definitely still be here! We have

```ts
type Thing = { a: string, b: string };
```

so for example, `T = { a: "x" | "y" | "z", b: "hello" | "goodbye" }` is a subtype of `Thing`, but you can't allow `y = x`, since `x` could be e.g. `{ a: "foo", b: "bar" }` which is definitely not a `Readonly<T>`.

I think all of the other errors in this file should also stay, but have not looked carefully at them; none of them are using `Partial` so the same reasoning should apply in all cases: the errors are correct.

Reply:
yup, thanks for explaining this!